docker-compose.yml

```diff
@@ -429,10 +415,126 @@ services:
       - 127.0.0.1:8093:8080
     networks:
       placeos:
-        ipv4_address: 172.31.231.19
     volumes:
       - type: volume
         source: rethink-data
         target: /data/rethinkdb_data
     environment:
       TZ: $TZ
+
+  # Aggregates logs and forwards them to Elasticsearch.
+  logstash:
+    image: docker.elastic.co/logstash/logstash-oss:${ELASTIC_VERSION:-7.6}
+    restart: always
+    container_name: logstash
+    hostname: logstash
+    expose:
+      - 12201/udp
+      - 5044
+    ports:
+      - "127.0.0.1:12201:12201/udp"
+    networks:
+      placeos:
+    volumes:
+      - ./config/logstash/config:/config
+      - ./config/logstash/patterns:/opt/logstash/extra_patterns
+    restart: always
+    labels:
+      container_group: logging
+    logging:
+      driver: json-file
+      options:
+        max-size: 99m
+        labels: container_group
+    command: logstash -f /config
+
+  # Run 'docker-compose run --rm validate-logstash-config' to quickly check the logstash config.
+  validate-logstash-config:
+    container_name: validate-logstash
+    image: docker.elastic.co/logstash/logstash-oss:${ELASTIC_VERSION:-7.6}
+    volumes:
+      - ./config/logstash/config:/config
+    command: logstash -t -f /config
+
+  # Sends all container json-file logs to logstash
+  logspout:
+    image: vincit/logspout-gelf
```

Comment:
This looks to be an oldish container, are there any equivalent containers that are more consistently updated?

Reply:
I'll have a look into that

Reply:
There is gliderlabs/logspout which is more active. 
Looks like we'll need to build our own container, but it does have `logspout-logstash` module available which is probably worth switching to.
https://github.com/gliderlabs/logspout#modules

I'll make a separate issue to look into that.

Reply:
https://github.com/place-labs/partner-environment/issues/36